### PR TITLE
Features/style & extra functionalities

### PR DESCRIPTION
**Pull request by Alzubair98**
all the requirements are done 👍 

(i used the google developers tool with iPhone 12.)

1-Select the API you are going to use.
2-Build files structure for your React app.
3-Prepare routes and navigation in your app.
4-Make sure that a user can display a list of items and filter them. Data should come from the API.
5-Create the tests for the application
6-Style your components to match the design provided.
7-Deploy the project and test for final details.
8-Record a video for your project.
9-Create a good README and PR description.

### CODE REVIEW

File: src/Redux/phones/phones.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const LOADING = 'LOADING';
+// const LOAD_IMG = 'LOADIMG';
```
Comment (tufoinnkuo10) [inline src/Redux/phones/phones.js]: - [x] please remove commented code for clean code purposes

File: src/Redux/phones/phones.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const url = 'https://zoo-animal-api.herokuapp.com/animals/rand/10';
+// const firstPageImg = 'https://zoo-animal-api.herokuapp.com/animals/rand';
```
Comment (tufoinnkuo10) [inline src/Redux/phones/phones.js]: - [x] kindly remove commented code

File: src/Redux/phones/phones.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }));
+  return data;
+});
+
+// export const loadimg = createAsyncThunk(LOAD_IMG, async () => {
+//   const response = await axios.get(firstPageImg).catch((error) => {
+//     console.log('Error', error);
+//   });
+//   return response.data;
+// });
```
Comment (tufoinnkuo10) [inline src/Redux/phones/phones.js]: - [x] this as well

File: README.md [hunk 4]
```diff
@@ -24,7 +32,7 @@
 
 ## Set Up Guide :page_facing_up:
 
-1. Use the following link for "$ git clone" the [repo](https://github.com/Alzubair98/REACT-GROUP-PROJECT.git) and cd REACT-GROUP-PROJECT
+1. Use the following link for "$ git clone" the [repo](https://github.com/Alzubair98/react-capstone-project.git) and cd REACT-GROUP-PROJECT
```
Comment (tufoinnkuo10) [inline README.md:38]: - [x] please ensure that the acknowledgment section gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 5]
```diff
@@ -1,14 +1,22 @@
...
 ![React](https://img.shields.io/badge/react-%2320232a.svg?style=for-the-badge&logo=react&logoColor=%2361DAFB)
 ![Redux](https://img.shields.io/badge/redux-%23593d88.svg?style=for-the-badge&logo=redux&logoColor=white)
 ![Webpack](https://img.shields.io/badge/webpack-%238DD6F9.svg?style=for-the-badge&logo=webpack&logoColor=black)
 ![Jest](https://img.shields.io/badge/-jest-%23C21325?style=for-the-badge&logo=jest&logoColor=white)
 
-# Space Traveler's Hub :rocket: :astronaut:
+# Animal Album
+
+> A Website App built with React and Redux for a company that provides commercial and scientific animals informations. The application fetches the data from the zoo animal API and allows users to see specific animal information.
+
```
Comment (tufoinnkuo10) [inline README.md:11]: - [x] please ensure that the application is either deployed using Heroku, Netlify or GitHub Pages and A link to the deployed project is present in the README

File: src/App.css [hunk 6]
```diff
@@ -1,38 +1,193 @@
...
+  margin: 0;
+  padding: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.Nav-bar,
+#footer {
+  display: flex;
```
Comment (tufoinnkuo10) [inline src/App.css:10]: - [x] I used iPhone 12 to test as you specified. Kindly fix the home link right padding, the images on the right need to be adjusted a little as well. Nice work
Screenshot (375)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Alzubair98 

great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well implemented with home and details page
✅ test passing
✅ video demonstrated well

## Required Changes ♻️
Check the comments under the review

- [ ]Great job with the video. but you were just showing the different images.  please make another short brief video that contains a highlight about a piece of the code that you find interesting or you are proud of making.


tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css (hunk 6 above)
Comment (Alzubair98) [reply src/App.css:10]: @tufoinnkuo10 
okay that's weird now 
kindly look at my google dev tools it appears like that 
screen shot

also you can check the loom video its working correctly 👍

File: README.md (hunk 5 above)
Comment (Alzubair98) [reply README.md:11]: done

File: README.md (hunk 4 above)
Comment (Alzubair98) [reply README.md:38]: done

File: src/Redux/phones/phones.js (hunk 3 above)
Comment (Alzubair98) [reply src/Redux/phones/phones.js]: done

File: src/Redux/phones/phones.js (hunk 2 above)
Comment (Alzubair98) [reply src/Redux/phones/phones.js]: done

File: src/Redux/phones/phones.js (hunk 1 above)
Comment (Alzubair98) [reply src/Redux/phones/phones.js]: done

**Review by addod19 — APPROVED** [review]
### Hi @Alzubair98, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] README is descriptive with detailed acknowledgements :+1: awesome
- [x] Commented code removed :rocket: 
- [x] The UI looks good on my iphone 7+ :100: 


### Optional
- [x] Kindly fix the linter errors, because this is not a first review I shall pass it out as optional, kindly fix it before proceeding

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**